big_tests/rebar.config

```diff
@@ -14,14 +14,14 @@
         {erlsh, {git, "https://github.com/proger/erlsh.git", {ref, "4e8a107"}}},
         {jiffy, "1.0.8"},
         {proper, "1.4.0"},
+        {gun, "1.3.2"},
```

Comment:
Why 1.3.2? There's 1.3.3 available also 🤔

Reply:
Cuz this is the same version what used in `escalus` with ref `ac5e813ac96c0cdb5d5ac738d63d992f5f948585`. Here should be the same deps version as in `escalus`. When `escalus` will be updated, also wil be updated `gun` and `bbmustache`.

Reply:
All right, I'm fine with that 🙂